docs/core-blockchain-concepts/proof-of-stake/stake-enabled-height.md

```diff
@@ -1,3 +1,26 @@
 # <img class="dcr-icon" src="/img/dcr-icons/TicketLive.svg" /> Stake Enabled Height
 
 ---
+
+Stake Enabled Height (SEH) is one of the [Blockchain Parameters](../blockchain-parameters.md).
+
+SEH describes the height at which the first Proof-of-Stake ticket could possibly mature.
```

Comment:
I wasn't sure what "the first Proof-of-Stake" ticket referred to until reading the rest of the paragraph. I think this could be clearer if we explicitly spell out what "the ticket" means. From my understanding, this could be something like, "..the block height at which new coins created from coinbase and stakebase transactions can be staked in a mature ticket. This is a function of the CoinbaseMaturity (the number of blocks that must be added after coins are created before they can be spent), and the TicketMaturity (the number of blocks that must be created before a newly bought ticket is eligible to vote). For instance, the first coins..."

Reply:
SEH does not refer to a **duration**, it refers to a specific **height**. (256 on mainnet)

https://explorer.dcrdata.org/block/256
https://explorer.dcrdata.org/block/257

You can see here how 256 included zero ticket purchases, and the first block after that included 18. This is because there were no spendable DCR on the network before block 257.


Maybe these links should be included? Do you think it helps illustrate the point?

On a similar vein for SEH, you can see that the block before SEH included no votes, the first block after SEH included 5

https://explorer.dcrdata.org/block/4095
https://explorer.dcrdata.org/block/4096

Reply:
>SEH does not refer to a duration, it refers to a specific height. (256 on mainnet)

Well, duration and height are related (by ~5 min). I tend to think of blocks in terms of time, as will others. The text as it's worded now seems clear though. I don't think we need links to the block explorer (though open to it if you think it adds clarity). My only nit is with the first sentence, 

>"SEH describes the height at which the first Proof-of-Stake (PoS) ticket could possibly mature."

While this is correct, and sets up the example of the premine coins (which does clearly convey what the SEH is), SEH does not _only_  describe the height at which the _first_ ticket could mature, which is somewhat implied by this sentence. It's all tickets. Also, 'could possibly' implies variability, but the `CoinbaseMaturity` + `TicketMaturity` will always be the same, no? Maybe just say, "SEH describes the height at which Proof-of-Stake (PoS) tickets mature (are eligible to be called to vote).".

Reply:
> > SEH does not refer to a duration, it refers to a specific height. (256 on mainnet) 
>
> Well, duration and height are related (by ~5 min). I tend to think of blocks in terms of time, as will others.

To clarify, we are **only** talking about one block height, the one after height 256. [This one](https://explorer.dcrdata.org/block/257). We are not talking about durations, or windows of 256 blocks. Literally just one block (or more specifically, its height).

> SEH does not only describe the height at which the first ticket could mature

SEH **does exactly** describe the height at which the first ticket could possibly mature.

>  Also, 'could possibly' implies variability, but the CoinbaseMaturity + TicketMaturity will always be the same, no?

The "possibly" part is because its not guaranteed. If no tickets are purchased, no tickets will mature on SEH.